app/src/main/java/com/kodeco/android/coordplot/SplashScreen.kt

```diff
@@ -0,0 +1,46 @@
+package com.kodeco.android.coordplot
+
+import android.view.animation.OvershootInterpolator
+import androidx.compose.animation.core.Animatable
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.scale
+import androidx.compose.ui.res.painterResource
+import androidx.navigation.NavController
+import kotlinx.coroutines.delay
+
+@Composable
+fun SplashScreen(navController: NavController) {
+    val scale = remember {
```

Comment:
Great work here on this SplashScreen! This is definitely going above and beyond! The animation looks great! 🎉